deploy/sdk/src/dynamo/sdk/core/lib.py

```diff
@@ -0,0 +1,76 @@
+import os
+from typing import Any, Dict, Optional, Type, TypeVar, Union
+
+from fastapi import FastAPI
+
+from dynamo.sdk.core.protocol.interface import (
+    DependencyInterface,
+    DeploymentTarget,
+    DynamoConfig,
+    ServiceConfig,
+    ServiceInterface,
+)
+
+T = TypeVar("T", bound=object)
+
+#  Note: global service provider.
+# this should be set to a concrete implementation of the DeploymentTarget interface
+_target: DeploymentTarget
+
+
+DYNAMO_IMAGE = os.getenv("DYNAMO_IMAGE", "dynamo:latest-vllm")
```

Comment:
What if DYNAMO_IMAGE doesn't exist?

Reply:
I think that there should be a check here -- I agree that the user should explicitly pass it in, but we should add a check/message if the user doesn't, asking them to pass it in. And give `dynamo:latest-vllm` as an example/link them to the docs where they can find/make a dynamo image. Rather than silently falling back to a default that may or may not exist